src/containers/PipelineRun/PipelineRun.js

```diff
@@ -284,6 +286,8 @@ export /* istanbul ignore next */ function PipelineRunContainer(props) {
         />
       )}
       <PipelineRun
+        enableLogAutoScroll={enableLogAutoScroll}
```

Comment:
no need to accept this as a prop, just set the attribute here
```jsx
<PipelineRun
  enableLogAutoScroll
  enableLogScrollButtons
  error={error}
…
```


same for the TaskRun container